### ISSUE TITLE
Cannot reconnect at all

### ISSUE THREAD

**Issue by tiotrom** [RetroShare/RetroShare#1582]
I use Manjaro Gnome + latest retroshare.
I am behind a VPN with port forwarding. If I normally open retroshare it works with the default settings. If I then close it and reopen and select Manually Forwarded Port and add the right one, it fails to connect (says "NAT DHT disabled" and "DHT: No peer found in DHT"). Now, I try to put the settings back and select the default Automatic UPnP. I wait a long time and does not connect back. it says "NAT: DHT Disabled and Firewalled" and "DHT: No peer found in DHT".

Basically I am unable to reconnect again.

If I wipe all the config files and start as brand new, it will work though.

Any idea what is happening?

**Comment by felisucoibi** [comment]
Maybe your friends are offline, how many of them you have?

**Comment by tiotrom** [comment]
4-5 friends. No they are not offline. And I can reconnect if I wipe the config files and start as "brand new".....

**Comment by defnax** [comment]
@csoler close

**Comment by defnax** [comment]
@thunder2 close
